server/incore-common/src/main/java/edu/illinois/ncsa/incore/common/auth/Authorizer.java

```diff
@@ -193,46 +195,37 @@ public boolean canUserWriteMember(String username, String memberId, List<Space>
     /**
      * This method determines if a user is an admin or not.
      *
-     * @param username string name of the user
+     * @param userGroups list of strings of groups user belongs to
      * @return boolean - true if the user is admin
      */
-    public boolean isUserAdmin(String username) {
-        try {
-            LdapClient ldapClient = getLdapClient();
-            Set<String> userGroups = ldapClient.getUserGroups(username);
-
-            return userGroups.contains(System.getenv("AUTH_LDAP_ADMINS"));
-        } catch (Exception e) {
-            logger.error(e);
-            throw new IncoreHTTPException(Response.Status.INTERNAL_SERVER_ERROR, "Internal server error.");
-        }
+    public boolean isUserAdmin(List<String> userGroups) {
+        return (userGroups.contains("incore_admin") || userGroups.contains("incore_ncsa"));
```

Comment:
on a second thought... might be safer to pass it in as environment variable with default set to "incore_admin" and "incore_ncsa" just in case in the future we might change those group name

Reply:
So, should I just revert it back to the original env var name? because the name of the env var can be same and the value can change right?

Reply:
That's a good idea!